### PR TITLE
app-misc/piper: Remove flake8 test

### PR DESCRIPTION
**Pull request by kwhat**
Closes: https://bugs.gentoo.org/807205
Signed-off-by: Alexander Barker <alex@1stleg.com>

### CODE REVIEW

File: app-misc/piper/piper-0.5.1-r2.ebuild [hunk 1]
```diff
@@ -40,6 +40,8 @@ DEPEND="
 	virtual/libudev
 "
 
+PATCHES=( "${FILESDIR}"/${P}-disable-flake8-linting.patch )
```
Comment (ionenwks) [inline app-misc/piper/piper-0.5.1-r2.ebuild:43]: nitpick but even for 1 patch I tend to prefer patch on its own line (makes it quicker to add/remove patches without restyling everytime), aka:
```suggestion
PATCHES=(
	"${FILESDIR}"/${P}-disable-flake8-linting.patch
)
```

File: app-misc/piper/files/piper-0.5.1-disable-flake8-linting.patch [hunk 2]
```diff
@@ -0,0 +1,14 @@
+--- piper-0.5.1/meson.build	2020-06-22 01:07:53.000000000 -0700
++++ piper-0.5.1/meson.build	2021-08-08 11:46:32.984756474 -0700
+@@ -119,11 +119,3 @@
+                configuration: config_piper_devel)
+ 
+ meson.add_install_script('meson_install.sh')
+-
+-flake8 = find_program('flake8', required: false)
+-if flake8.found()
```
Comment (ionenwks) [inline app-misc/piper/files/piper-0.5.1-disable-flake8-linting.patch:9]: If making patches that are never going to be upstreamed, you can (optionally) cheat to have shorter patches by (for example) using `if false`

But it's fine if you prefer it like that too.

**Review by ionenwks — COMMENTED** [review]
Commit message seem to have everything (including close/sign-off) is inside the summary instead of the message body.

File: app-misc/piper/files/piper-0.5.1-disable-flake8-linting.patch (hunk 2 above)
Comment (kwhat) [reply app-misc/piper/files/piper-0.5.1-disable-flake8-linting.patch:9]: This only results in one line saved for this particular patch, however, I will keep it in mind for larger patches.

**Pull request closed without merging**